src/pages/sihyun/Login/Login.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+
+const Login = () => {
+  const navigate = useNavigate();
+  const [user, setUser] = useState({
+    email: '',
+    password: '',
+  });
+  const emailRegEx =
+    /^[A-Za-z0-9]([-_.]?[A-Za-z0-9])*@[A-Za-z0-9]([-_.]?[A-Za-z0-9])*\.[A-Za-z]{2,3}$/;
+
+  const updateForm = e => {
+    const updateUser = { ...user, [e.target.name]: e.target.value };
```

Comment:
👍좋습니다!
event객체에 대한 구조분해 할당이 더해지면 훨씬 좋은코드가 되겠네요!
```suggestion
    const {name, value} = e.target
    const updateUser = { ...user, [name]: value };
```